doc/source/user_guide/numpyro_refitting_xr_lik.ipynb

```diff
@@ -13,7 +13,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Below there is one example of `SamplingWrapper` usage for NumPyro."
+    "Below there is an example of `SamplingWrapper` usage for NumPyro."
```

Comment:
Can you please add the link to NumPyro docs? 
"[NumPyro] (https://pyro.ai/numpyro/)"